sasmodels/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sasmodels" %}
+{% set version = "0.98" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 151a0e3d75c1cf42df8126a3fa919e988b0f332051e876383739d50034716373
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy
+    - scipy
+    - bumps
+    - pyopencl
```

Comment:
It seems that `bump` and `pyopencl` are only required if some [`extras`](https://github.com/SasView/sasmodels/blob/8803a3853d8388dbec0845dd84f394341fb00041/setup.py#L69) are installed, which you are currently not doing. Please decide what you want :)

Also, there seems to be a [Windows-only dependency](https://github.com/SasView/sasmodels/blob/8803a3853d8388dbec0845dd84f394341fb00041/setup.py#L34) that needs to be added.

Reply:
> It seems that bump and pyopencl are only required if some extras are installed, which you are currently not doing. Please decide what you want :)

Does it matter that I am not modifying the `build.script` command to add the extras? Pip won't be used to install the extra packages anyway right?

Reply:
The `tinycc` windows dependency is not installable via conda at the moment. It isn't necessary anyway if the system has OpenCL support.